sdk/messaging/azservicebus/internal/stress/tests/constant_detachment.go

```diff
@@ -30,19 +32,27 @@ func ConstantDetachment(remainingArgs []string) {
 	sender, err := client.NewSender(queueName, nil)
 	sc.PanicOnError("create a sender", err)
 
-	const maxMessages = 10000
+	const maxMessages = 20000
```

Comment:
Nit: Add comment on why this max was selected

Reply:
Done.